data_model/src/role.rs

```diff
@@ -49,18 +50,19 @@ pub struct Id {
     Debug,
     Display,
     Clone,
-    PartialEq,
-    Eq,
+    OrdEqHash,
```

Comment:
how come some Identifiables don't have `OrdEqHash`? Shouldn't all have it?

Reply:
What about trigger?

Reply:
you have full control over the macro. You could attach the `Filter` bound on the `Ord`, `Eq`, `Hash` impls for `Trigger`

Reply:
Created an [issue](https://github.com/hyperledger/iroha/issues/2513).